src/api/modules/generate/index.js

```diff
@@ -0,0 +1,58 @@
+import co from 'co';
```

Comment:
It's a third party module: https://www.npmjs.com/package/co 
PS: it allows you to use `yield`